lib/sparrow/fcm/v1/token_bearer.ex

```diff
@@ -19,15 +19,25 @@ defmodule Sparrow.FCM.V1.TokenBearer do
   end
 
   @spec start_link(Path.t()) :: GenServer.on_start()
-  def start_link(google_json_path) do
-    json = File.read!(google_json_path)
+  def start_link(raw_fcm_config) do
+    json_list =
```

Comment:
`[]` is valid JSON but it is not the most beautiful one ;)

Reply:
What do you mean? Which type of JSON is "the most beautiful one" and how is that relevant / objective? ;P

Reply:
@rslota  https://stackoverflow.com/questions/19623339/is-a-list-array-valid-json